fairscale/nn/misc/flatten_params_wrapper.py

```diff
@@ -70,6 +70,9 @@ def __new__(cls, params: Sequence[nn.Parameter], requires_grad: bool = True) ->
     def __init__(self, params: Sequence[nn.Parameter], requires_grad: bool = True):
         """Initialize the _param_numels and _param_shapes lists."""
         self._param_numels = [p.numel() for p in params]
+        self._param_require_allreduce = [
```

Comment:
Perhaps more descriptive name is _param_require_tp_allreduce?

Reply:
Thanks for the suggestion! I have made the modification.